lock-api-objects/src/main/java/com/palantir/lock/v2/StartTransactionRequestV4.java

```diff
@@ -33,7 +34,7 @@ public interface StartTransactionRequestV4 {
 
     static StartTransactionRequestV4 createForRequestor(UUID requestorUuid, int numTransactions) {
         return ImmutableStartTransactionRequestV4.builder()
-                .requestId(UUID.randomUUID())
+                .requestId(UniqueIds.randomUuidV4())
```

Comment:
I think pseudorandom should do? This is the ID for the lease token we use to lock the immutable timestamp, so it should be the same as those

Reply:
thanks, updated